apps/console/src/features/organizations/pages/organization-edit.tsx

```diff
@@ -87,7 +110,10 @@ const OrganizationEditPage: FunctionComponent<OrganizationEditPagePropsInterface
             return;
         }
 
-        setIsAuthorizedOrganization(authorizedOrganizationList.organizations?.length === 1);
+        const isOrgAuthorized: boolean = !!authorizedOrganizationList.organizations
+            ?.find((org: OrganizationInterface) => org.id === organization?.id);
```

Comment:
We can avoid using double negation by using the `Array.some` function.

```suggestion
        const isOrgAuthorized: boolean = authorizedOrganizationList?.organizations
            ?.some((org: OrganizationInterface) => org.id === organization?.id);
```

Reply:
Using double negation is preferred here because `authorizedOrganizationList.organizations` can be `undefined` which would make `isOrgAuthorized` variable `undefined` instead of a boolean value.